doc/tutorials/enable_s5.rst

```diff
@@ -205,3 +205,45 @@ How to test
 
       # acrnctl list
       vm1		stopped
+
+System Shutdown
+***************
+
+Using a coordinating script, ``misc/life_mngr/s5_trigger.sh``, in conjunction with
+the lifecycle manager in each VM, graceful system shutdown can be performed.
+
+.. note:: Please install ``s5_trigger.sh`` manually to root's home directory.
+
+   .. code-block:: none
+
+      # sudo install -p -m 0755 -t ~root misc/life_mngr/s5_trigger.sh
```

Comment:
```suggestion
      $ sudo install -p -m 0755 -t ~root misc/life_mngr/s5_trigger.sh
```

Reply:
I've never seen this thing: `~root`, is that a shortcut to reference home directory of the root user?

Reply:
> I've never seen this thing: `~root`, is that a shortcut to reference home directory of the root user?

Yes, that is correct.

Reply:
Thanks, I've learned something new today :-)